packages/ag-charts-community/src/options/chart/chartOptions.ts

```diff
@@ -147,6 +154,8 @@ export interface AgBaseThemeableChartOptions<TDatum = any> {
     zoom?: AgZoomOptions;
     /** Configuration for the Navigator. */
     navigator?: AgNavigatorOptions;
+    /** Configuration for synchronizing multiple charts. */
+    sync?: AgChartSyncOptions;
```

Comment:
Could we hold off adding this to the options contract until we have done a few iterations with David and agreed the options contract?

Reply:
I'll comment it out